qa/workunits/mon/pg_autoscaler.sh

```diff
@@ -45,6 +45,35 @@ ceph osd pool set b pg_autoscale_mode on
 # get num pools again since we created more pools
 NUM_POOLS=$(ceph osd pool ls | wc -l)
 
+# get profiles of pool a and b
+PROFILE1=$(ceph osd pool autoscale-status | grep 'a' | grep -o -m 1 'scale-up\|scale-down' || true)
+PROFILE2=$(ceph osd pool autoscale-status | grep 'b' | grep -o -m 1 'scale-up\|scale-down' || true)
+
+# evaluate the default profile a
+if [[ $PROFILE1 = "scale-up" ]]
+then
+  echo "Success: pool a PROFILE is scale-up"
+else
+  echo "Error: a PROFILE is scale-down"
+  exit 1
+fi
+
+# evaluate the default profile of pool b
+if [[ $PROFILE2 = "scale-up" ]]
+then
+  echo "Success: pool b PROFILE is scale-up"
+else
+  echo "Error: b PROFILE is scale-down"
+  exit 1
+fi
+
+# This part of this code will now evaluate the accuracy of
+# scale-down profile
+
+# change to scale-down profile
+
+ceph osd pool set autoscale-profile scale-down 
```

Comment:
In https://pulpito.ceph.com/ksirivad-2021-11-19_15:13:16-rados:singleton:all:pg-autoscaler.yaml-wip-autoscale-profile-scale-up-default-distro-basic-smithi/, why do we see "already a scale-down"? Shouldn't we do a similar check for autoscale-status here like above?

```
2021-11-19T15:30:30.855 INFO:tasks.workunit.client.0.smithi116.stdout:Success: pool b PROFILE is scale-up
2021-11-19T15:30:30.855 INFO:tasks.workunit.client.0.smithi116.stderr:+ echo 'Success: pool b PROFILE is scale-up'
2021-11-19T15:30:30.856 INFO:tasks.workunit.client.0.smithi116.stderr:+ ceph osd pool set autoscale-profile scale-down
2021-11-19T15:30:31.221 INFO:tasks.workunit.client.0.smithi116.stderr:autoscale-profile is already a scale-down!
```